### DISCUSSION TITLE
Source Control Empty

### DISCUSSION THREAD

**Discussion by markpanado** [blinksh/blink#1548] [Ideas]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Describe the bug

- Open vscode with command `code remoteserver:/example/path`
- `/example/path` is a git repository, can confirm in another blink terminal tab
- Added changes in `/example/path` via vscode
- No changes log in Source Control tab
- running `git status` in another blink terminal shows changes

### Expected behavior
- Source Control section should show the changes

### Screenshots
In blink terminal:
![](https://i.imgur.com/TbgfXHl_d.png?maxwidth=640&shape=thumb&fidelity=medium)

In blink vscode:
![](https://i.imgur.com/xhwu6sm_d.png?maxwidth=640&shape=thumb&fidelity=medium)

**Comment by carloscabanero** [comment]
This is currently not supported by VS Code itself. We do not have access to the remote file system in that way, but maybe in the future. We will keep it as an idea within our Discussions in case someone may know.

**Reply by markpanado** [reply]
Noted on this, Carlos. Thank you!

**Reply by pcho** [reply]
> Noted on this, Carlos. Thank you!

If you install vscode-server from Microsoft on your remote host it will work. You can find how to do it here: https://github.com/microsoft/vscode/issues/135856#issuecomment-1170091265

Or use below scrip to download, extract, and run vscode-server. 

```shell
wget https://az764295.vo.msecnd.net/insider/$(curl https://update.code.visualstudio.com/api/commits/insider/server-linux-x64-web | cut -d '"' -f 2)/vscode-server-linux-x64-web.tar.gz
mkdir "$HOME"/vscode
tar -C "$HOME"/vscode -zxvf vscode-server-linux-x64-web.tar.gz
./vscode/vscode-server-linux-x64-web/bin/code-server-insiders --accept-server-license-terms --without-connection-token
```

Then you can connect as usual using `code remotehost`.

Here’s an example:
![5EBDE630-27F3-48F2-B924-828E3CB2DD8E](https://user-images.githubusercontent.com/41922527/185764146-b467512d-594b-4f8f-8a5b-471b6a3c566a.png)
